.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+on:
+  push:
+    branches:
+      - master
+      - release
```

Comment:
Why just these branches? Also, there's no `release` branch. :upside_down_face: The current Travis setup runs on all branches + all PRs, and it seems like this should do the same...

Reply:
Whoops, this was copy/paste from another workflow. Fixed to be consistent with https://github.com/nextstrain/augur/blob/master/.github/workflows/ci.yaml.